### PR TITLE
NickAkhmetov/Improve push script specificity

### PR DESCRIPTION
**Pull request by NickAkhmetov**
This PR adjusts the specificity of the push script to only target the files that are intended to be committed during the release process.

### CODE REVIEW

File: etc/build/push.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ if ls CHANGELOG-*.md; then
   ) > CHANGELOG.md.new
   mv CHANGELOG.md.new context/app/markdown/CHANGELOG.md
   git rm CHANGELOG-*.md
-  git add .
+  git add context/app/markdown/CHANGELOG.md
```
Comment (john-conroy) [inline etc/build/push.sh:52]: We also have to commit the file deletions at the project root?

File: etc/build/push.sh [hunk 2]
```diff
@@ -34,7 +34,7 @@ echo "Version: $VERSION"
 
 ./grab-dependencies.sh
 
-git add .
+git add context/package*.json
```
Comment (john-conroy) [inline etc/build/push.sh:37]: I don't suggest doing this here, but we could consider pulling some of the git version/git tag logic out of this script and into package.json? https://docs.npmjs.com/cli/v7/commands/npm-version details some of the options.

**Review by john-conroy — COMMENTED** [review]
Thanks!

File: etc/build/push.sh (hunk 1 above)
Comment (NickAkhmetov) [reply etc/build/push.sh:52]: `git rm` stages the removals; I just tested this locally to be sure:

```
(portal) nikolay@NikolayPCWin:~/projects/hubmap/portal-ui$ git rm example-app.conf 
rm 'example-app.conf'
(portal) nikolay@NikolayPCWin:~/projects/hubmap/portal-ui$ git status
On branch nickakhmetov/hmp-348/misc-dependency-updates
Your branch is up to date with 'origin/nickakhmetov/hmp-348/misc-dependency-updates'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        deleted:    example-app.conf
```

File: etc/build/push.sh (hunk 2 above)
Comment (NickAkhmetov) [reply etc/build/push.sh:37]: I think that'd be a good idea! I made a ticket: https://hms-dbmi.atlassian.net/browse/HMP-410

**Review by john-conroy — APPROVED** [review]

**Pull request merged**